src/main/steps/home/get.ts

```diff
@@ -78,6 +79,9 @@ const applicant1RedirectPageSwitch = (caseState: State, userCase: Partial<Case>,
     case State.AwaitingGeneralConsideration: {
       return HUB_PAGE;
     }
+    case State.ConditionalOrderDrafted: {
+      return READ_THE_RESPONSE;
```

Comment:
would you know run through the steps to find out what part of the CO they're on?

Reply:
i believe at this point itll end on the hub page considering the hub page and the read the response page are just continue buttons.